recipes/pathlib-abc/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "pathlib_abc" %}
```

It is more common to use a dash here.
```suggestion
{% set name = "pathlib-abc" %}
```